### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by hedvigoscar**
🏡

### CODE REVIEW

File: app/src/main/java/com/hedvig/android/app/LoggedInActivity.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import com.hedvig.android.owldroid.util.extensions.view.doOnLayout
+
+class LoggedInActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val view = FrameLayout(this)
+
+        view.addView(layoutInflater.inflate(R.layout.logged_in_host, view, false))
+        view.doOnLayout {
+            val bottomTabs = view.findViewById<BottomNavigationView>(R.id.bottomTabs)
```
Comment (FredrikAreschoug) [inline app/src/main/java/com/hedvig/android/app/LoggedInActivity.kt]: Just so I know why do we need to do this on `doOnLayout`?

File: owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+
+        val content = layoutInflater.inflate(
+            R.layout.dashboard_footnotes,
+            additionalInformation.expandedContentContainer,
+            false
+        )
+
+        content.totalCoverageFootnote.text = interpolateTextKey(
+            resources.getString(R.string.DASHBOARD_INSURANCE_AMOUNT_FOOTNOTE),
+            "AMOUNT" to if (isStudentInsurance(insuranceType)) "200 000" else "1 000 000"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt]: Move amounts strings to constants

File: owldroid/src/main/res/layout/peril_category_view.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        android:paddingTop="@dimen/base_margin_triple"
+        android:paddingBottom="@dimen/base_margin_triple"
+        android:paddingStart="@dimen/base_margin_double"
+        android:paddingEnd="@dimen/base_margin_double"
+        tools:layout_height="120dp">
+
+        <ImageView
+            android:id="@+id/catIcon"
+            android:contentDescription="@null"
+            android:layout_width="32dp"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/peril_category_view.xml]: Extract dimmens

File: owldroid/src/main/res/layout/peril_category_view.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            android:layout_height="32dp"
+            app:layout_constraintBottom_toTopOf="@id/perilsContainer"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:src="@drawable/ic_family" />
+
+        <LinearLayout
+            android:orientation="vertical"
+            android:layout_marginStart="16dp"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/peril_category_view.xml]:
```suggestion
            android:layout_marginStart="@dimen/base_margin_double"
```

File: owldroid/src/main/res/layout/peril_category_view.xml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+            android:layout_height="match_parent"
+            app:layout_constraintBottom_toTopOf="@id/perilsContainer"
+            app:layout_constraintStart_toEndOf="@id/catIcon"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <TextView
+                android:id="@+id/categoryTitle"
+                android:textSize="16sp"
+                android:textColor="@color/off_black_dark"
+                android:fontFamily="@font/circular_bold"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/peril_category_view.xml]:
```suggestion
                style="@style/Body.Regular"
```

File: owldroid/src/main/res/layout/peril_category_view.xml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                tools:text="Jag och min familj" />
+
+            <TextView
+                android:id="@+id/categorySubtitle"
+                android:fontFamily="@font/circular_book"
+                android:textSize="14sp"
+                android:textColor="@color/off_black"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/peril_category_view.xml]:
```suggestion
                style="@style/Body"
```

File: owldroid/src/main/res/layout/fragment_dashboard.xml [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="@font/circular_book"
+                    android:text="@string/DASHBOARD_INSURANCE_ACTIVE"
+                    android:textColor="@color/off_black_dark"
+                    android:textSize="14sp" />
+
+                <ImageView
+                    android:layout_width="16dp"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/fragment_dashboard.xml]: Extract to dimmens

File: owldroid/src/main/res/layout/fragment_dashboard.xml [hunk 8]
```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/fragment_dashboard.xml]: Look at using text styles in this file

File: app/src/main/java/com/hedvig/android/app/LoggedInActivity.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/android/app/LoggedInActivity.kt]: Just some weird stuff with React-Native-Navigation. We'll get rid of this when we get rid of RNN

**Pull request merged**